### PR TITLE
chore(deps): upgrade most production deps

### PR DESCRIPTION
**Pull request by AviVahl**
adjusted `chalk` import/initialization to new major

### CODE REVIEW

File: src/instances.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import chalk, { Chalk } from 'chalk';
+import chalk = require('chalk');
```
Comment (johnnyreilly) [inline src/instances.ts:1]: Could we avoid using `require` imports please?

Comment (AviVahl) [reply src/instances.ts]: done.

**Comment by johnnyreilly** [comment]
Awesome. Could you increment the version the `package.json` and add an entry to the `CHANGELOG.md` please?

**Comment by AviVahl** [comment]
Done. I avoided adding the `thanks @<myself>` to the changelog. Otherwise, I get flooded with Dependabot notifications including that `@`. XD

**Comment by johnnyreilly** [comment]
I didn't think dependabot read `CHANGELOG.md` - could be wrong.

It's fine to not have your handle in there if that's the case. Please could you put "thanks Avi Vahl" instead then please? That's human readable and as such I'm fine with it.

**Comment by AviVahl** [comment]
Done.

**Comment by AviVahl** [comment]
Oh, and Dependabot picks up the release notes from the GitHub releases page. I already got spammed from a contribution to handlebars once, as they creditted my handle.

**Comment by johnnyreilly** [comment]
> Oh, and Dependabot picks up the release notes from the GitHub releases page

Interesting - I would have imagined it used the related commits and the authors there instead. Go figure

**Comment by johnnyreilly** [comment]
Shipped - thanks for the contribution!

**Comment by Stanzilla** [comment]
@AviVahl this broke https://github.com/TypeStrong/ts-loader/issues/1240

**Pull request merged**